pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -36,10 +36,11 @@ import (
 )
 
 const (
-	nodeShutdownReason          = "Shutdown"
-	nodeShutdownMessage         = "Node is shutting, evicting pods"
-	nodeShutdownNotAdmitMessage = "Node is in progress of shutting down, not admitting any new pods"
-	dbusReconnectPeriod         = 1 * time.Second
+	nodeShutdownReason             = "Terminated"
+	nodeShutdownMessage            = "Pod was terminated in response to node shutdown."
```

Comment:
(nit) I'd prefer to be clear that the shutdown was due but had not yet happened when the termination took place, eg:
```suggestion
	nodeShutdownMessage            = "Pod was terminated in response to imminent node shutdown."
```

Reply:
+1 makes sense, I like this wording.